public/app/features/explore/ExploreActions.tsx

```diff
@@ -65,6 +67,24 @@ export const ExploreActions = () => {
         });
       }
     } else {
+      // command palette doesn't know what pane we're in, only show option if not split and no datasource is mixed
+      const hasMixed = Object.entries(panes)
+        .map((pane) => pane[1])
+        .some((pane: ExploreItemState | undefined) => {
+          return pane?.datasourceInstance?.uid === MIXED_DATASOURCE_NAME;
+        });
+
+      if (config.featureToggles.correlations && !hasMixed) {
```

Comment:
Should we also check for permissions to create correlations?

Reply:
Good catch!